sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/ModelsLifecycleSyncSamples.java

```diff
@@ -0,0 +1,117 @@
+package com.azure.digitaltwins.core;
+
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.digitaltwins.core.helpers.ConsoleLogger;
+import com.azure.digitaltwins.core.helpers.SamplesArguments;
+import com.azure.digitaltwins.core.helpers.UniqueIdHelper;
+import com.azure.digitaltwins.core.implementation.models.ErrorResponseException;
+import com.azure.digitaltwins.core.models.ModelData;
+import com.azure.identity.ClientSecretCredentialBuilder;
+import org.apache.http.HttpStatus;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Creates a component with a random Id.
+ * Creates a new model with a random Id that uses the previously created component.
```

Comment:
Does the model _use_ the component, or _define_ the "model" for the component?

Reply:
technically they are both models, one is sort of like an interface that can be used in other models.